src/migration/migrator.ts

```diff
@@ -822,6 +822,18 @@ export interface MigratorProps {
    * Default is `name0.localeCompare(name1)`.
    */
   readonly nameComparator?: (name0: string, name1: string) => number
+
+  /**
+   * Whether to skip running the migration in a transaction. Only relevant to databases
+   * that support a transactional DDL (i.e. postgres and mssql).
+   * 
+   * When true, migrations will not be run in a transaction, even if the dialect 
+   * supports transactions in their DDL.
+   * 
+   * When false, migrations will be run in a transaction if that is supported by
+   * the dialect.
```

Comment:
```suggestion
   * When `true`, don't run migrations in transactions even if the dialect supports transactional DDL.
   *
   * Default is `false`.
   *
   * This is useful when some migrations include queries that would fail otherwise.
```